assets/src/hooks/useSocket.ts

```diff
@@ -1,11 +1,14 @@
 import { Socket } from "phoenix"
 import { useEffect, useState } from "react"
+import { UserToken } from "../skate"
 
-const useSocket = (): Socket | undefined => {
+const useSocket = (userToken?: UserToken): Socket | undefined => {
   const [socket, setSocket] = useState<Socket | undefined>(undefined)
 
   useEffect(() => {
-    const initialSocket = new Socket("/socket")
+    const initialSocket = new Socket("/socket", {
```

Comment:
I see you sending the token on socket connect, but not seeing you check it on socket connect. There should be a Guardian check on the `connect` callback in your socket code (`/channels/user_socket.ex` is the default, unless Skate changed it). See [here](https://github.com/mbta/signs_ui/blob/487d095cf72212d01ba031b7daa00fa21e490575/lib/signs_ui_web/channels/user_socket.ex#L23).

Reply:
Great catch! Added the authentication and tests proving it.